docs/README.md

```diff
@@ -29,6 +29,47 @@ async function main() {
 main();
 ```
 
+## Connection Management
+
+You are free to create your 'clients' like so:
+
+```typescript
+const client = new Client({
+  ...
+})
+await client.connect()
+```
+
+But for stronger management and scalability, you can use **pools**:
+```typescript
+import { Pool } from "https://deno.land/x/postgres@v0.4.0/mod.ts";
+import { PoolClient } from "https://deno.land/x/postgres@v0.4.0/client.ts";
+
+const POOL_CONNECTIONS = 50;
+const dbPool = new Pool({
+  user: "user",
+  password: "password",
+  database: "database",
+  hostname: "hostname",
+  port: 5432,
+}, POOL_CONNECTIONS);
+
+function runQuery (query: string) {
+  const client: PoolClient = await dbPool.connect();
+  const dbResult = await client.query(query);
+  client.release();
+  return dbResult
+}
+
+runQuery("SELECT * FROM users;");
+runQuery("SELECT * FROM users WHERE id = '1';");
+```
+
+This improves performance, as creating a whole new connection for each query can be an expensive operation.
+With pools, you can keep the connections open to be re-used when requested (`const client = dbPool.connect()`). So one of the active connections will be used instead  of creating a new one.
+
+The number of pools is up to you, but 50 is generally a good number, but this can differ based on how active your application is.
```

Comment:
Can't remember now 😕 Like you, i saw it somewhere 😄 I'll change it to 20 though 👍

Reply:
Changed it here: https://github.com/deno-postgres/deno-postgres/pull/150/commits/1d625184ee3e2d8f37feed26d1fc1a88dc2fb1b9, note when i say "I think ...", i refer to yourself

Reply:
ha! I just looked at a few applications and see 200 - so maybe I was confused (though I've no idea how well deno handles that many open). I'd probably say "many" applications (or even "most") rather than "small".

Will merge anyway. One thing I like to use sometimes is to have multiple query pools - that way longer-running queries can be put into a separate pool (and not slow down queries which are important to be fast).